src/ledger/NetworkConfig.h

```diff
@@ -47,6 +47,23 @@ struct MinimumSorobanNetworkConfig
     static constexpr uint32_t STARTING_EVICTION_LEVEL = 1;
 
     static constexpr uint32_t TX_MAX_CONTRACT_EVENTS_SIZE_BYTES = 200;
+
+    // SCP timing minimums
+    static constexpr uint32_t LEDGER_TARGET_CLOSE_TIME_MILLISECONDS = 4000;
```

Comment:
What's the rationale behind these ranges? (the CAP says they're approximate and untested). Asking because this limits what the network can do post p23 (and before any other protocol that may change these)

Reply:
IOW, should we make these more liberal for flexibility purposes? The actual upgrades will still be conservative and require global coordination of course.

Reply:
This came up in the CAP conversation, but we want these limits to be much narrower than most other config settings. Reason being, if we accidentally upgrade, or a subset of tier 1 does a "rogue upgrade", it's possible to brick the network, such that a follow up "fix" upgrade is not possible if we can't close ledgers. Unlike the other configs, it's pretty easy to reverse them, but there's not much you can do if you can't close blocks. These are all untested at this point, so we made the bounds very conservative to start. The idea is once we have more testing and perf work, if p24 is capable of lower blocktimes than 4 seconds, we lower the limit with the p24 release. 

For testing purposes, perhaps we can set different bounds via ifdef for ssc testing.

Reply:
Makes sense. It's fine to start conservative, but can you elaborate on the failure mode here (network unable to close)?

> I actually ended up changing these bounds slightly due to some unit tests values at the lower bounds. I think these are pretty safe for the release, but we'll need ssc testing for better understanding for actual upgrades and future bounds.

Can you elaborate on the specific changes? It sounds a bit suspicious to change prod values based on tests, unless the test pointed out that the initial values were wrong?